src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -117,23 +88,26 @@ private void initializeCommandTable() {
                         "Automatic",
                         "No command required"),
                 new CommandSummary(
-                        "Add New Customer",
+                        "Add New Client",
                         "add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOB> i/ <INCOME> "
-                                + "[t/ <TIER>] [rn/ <REMARK>]",
+                                + "[t/ <TIER>] [s/ <STATUS>] [r/ <REMARK>]",
                         "add n/ TAN LESHEW p/ 99007766 e/ mrtan@ntu.sg a/ com3 j/ doctor i/ 99999 "
-                                + "t/ gold rn/ got anger issue"),
-                new CommandSummary("Remove Old Customer",
+                                + "t/ gold s/ urgent r/ got anger issue"),
+                new CommandSummary("Remove Old Client",
                         "delete <INDEX>",
                         "delete 69"),
-                new CommandSummary("Edit Existing Customer",
+                new CommandSummary("Edit Existing Client",
                         "edit <INDEX> n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOB> i/ <INCOME> "
-                                + "[t/ <TIER>] "
+                                + "[t/ <TIER>] [s/ <STATUS>]"
                                 + "[rn/ <NEW REMARK>] [ra/ <REMARK TO BE APPENDED ONTO EXISTING ONE>]",
-                        "edit 69 n/ TAN LESHEW p/ 77337733 e/ mrtan@ntu.sg a/ COM3 j/ doctor i/ 1000000000"
-                                + "ra/ Specialist in eye care"),
-                new CommandSummary("Find a Customer by Details",
+                        "edit 69 n/ TAN LESHEW p/ 77337733 e/ mrtan@ntu.sg a/ COM3 j/ doctor i/ 1000000000 "
+                                + "t/ bronze s/ none ra/ Specialist in eye care"),
+                new CommandSummary("Find a Client by Details",
```

Comment:
Further from the UG, it uses "Filter Client List" for action column. Do you think that we should standardize?

Reply:
Yeah we should. I'll follow the UG